### PR TITLE
First occurence of 1 in a sorted array of 0s and 1s Docs

### PR DESCRIPTION
**Pull request by SAY-droid427**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

I have added an article for  finding the first occurence of 1 in a sorted array of 0s and 1s.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes https://github.com/TesseractCoding/NeoAlgo-Docs/issues/275

### CODE REVIEW

**Comment by SAY-droid427** [comment]
@HarshCasper Please review the PR and I cannot understand why the checks are failing. Would be grateful if you could provide some insights. thank you!

File: sidebars.js [hunk 1]
```diff
@@ -152,6 +153,7 @@ module.exports = {
                 "competitive-programming/anagram-numbers",
                 "competitive-programming/factorial-of-large-number",
                 "competitive-programming/max-number-from-array",
+                "competitive-programming/first-occurrence-1"
```
Comment (HarshCasper) [inline sidebars.js:156]: Missing comma here. Fix this and the build will pass

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: sidebars.js (hunk 1 above)
Comment (SAY-droid427) [reply sidebars.js]: Done!! Checks passing!!

**Comment by SAY-droid427** [comment]
@HarshCasper  Please review the pull request and merge

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**